packages/flutter/test/widgets/text_test.dart

```diff
@@ -310,6 +310,152 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('semantics label is in order when uses widget span', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: Text.rich(
+          TextSpan(
+            children: <InlineSpan>[
+              const TextSpan(text: 'before '),
+              WidgetSpan(
+                alignment: PlaceholderAlignment.baseline,
+                baseline: TextBaseline.alphabetic,
+                child: Semantics(label: 'foo'),
+              ),
+              const TextSpan(text: ' after'),
+            ],
+          ),
+        ),
+      ),
+    );
+    expect(tester.getSemantics(find.byType(Text)),
+        matchesSemantics(label: 'before \nfoo\n after'));
```

Comment:
here and elsewhere in this file: either leave this on one line or format as follows:

```suggestion
    expect(
      tester.getSemantics(find.byType(Text)),
      matchesSemantics(label: 'before \nfoo\n after'),
    );
```